### DISCUSSION TITLE
Why kcBodyClass class is not added in the ClassKey type definition of the TemplateProps.ts file of login theme?

### DISCUSSION THREAD

**Discussion by pranitkhadilkar7** [keycloakify/keycloakify#421] [General]
Why kcBodyClass class is not added in the ClassKey type definition of the TemplateProps.ts file of login theme?

**Comment by garronej** [comment]
Hi @pranitkhadilkar7,  

For no good reason, I just released a new version, 8.1.0, that addresses this:  

https://github.com/keycloakify/keycloakify/commit/de64deb5c5d25521fd606d5e87ff8706cbb49bc0  
https://github.com/keycloakify/keycloakify-starter/commit/0f469a1da13e060cf3b18652f6df11f9a82dc752  

Thanks for the heads up!
